app/controllers/container-type.js

```diff
@@ -18,6 +18,10 @@ export default Controller.extend({
   }).property('model.@each.name', 'search'),
 
   actions: {
+    clearSearch() {
+      this.set('searchValue', '');
```

Comment:
We're closer now, with these all having the same name, but it would be awesome to just move this stuff to a common component, instead of defining the same action a bunch of times. Could we pull this stuff into a common component, and use that component instead of partials? I believe partials are deprecated or at least recommended against.